cmd/watch-termination/main.go

```diff
@@ -31,6 +32,7 @@ func main() {
 func run() int {
 	terminationLog := flag.String("termination-log-file", "", "Write logs after SIGTERM to this file (in addition to stderr)")
 	terminationLock := flag.String("termination-touch-file", "", "Touch this file on SIGTERM and delete on termination")
+	processOverlapDetectionFile := flag.String("process-overlap-detection-file", "", "Presence of this file means the previous API servers was not properly terminated by kubelet and we had to fallback to slow run")
```

Comment:
"and we had to fallback to slow run" sounds strange.

Use the wording of the event.